src/status_im2/contexts/syncing/how_to_pair/style.cljs

```diff
@@ -0,0 +1,101 @@
+(ns status-im2.contexts.syncing.how-to-pair.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def scroll-view
+  {:flex             1
+   :background-color colors/neutral-95})
+
+(def container-outer
+  {:style {:background-color colors/neutral-95
```

Comment:
I don't think we want to refer to :style key in style files. This can just be"

```
(def container-outer
   {:background-color colors/neutral-95
           :justify-content  :space-between
           :align-items      :left
           :padding          16
           :border-radius    10})
           ```

Reply:
done